app/Http/Controllers/Settings/ProfileController.php

```diff
@@ -46,7 +46,23 @@ public function update(StoreSettingsProfile $request)
             ->with('success', trans('settings.profile.success'));
     }
 
-        /**
+    /**
+     * @param StoreSettingsProfile $request
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function saveBillingInfo(Request $request)
```

Comment:
The request should throw an error if the info is longer than 1024 characters, rather than just delete it without the user being warned that we removed content